doc/manual/source/developer_guide/DoingARelease.rst

```diff
@@ -4,7 +4,7 @@ Doing a Release
 ===============
 
 Periodically, the Enzo community creates an "official" release of the Enzo
-codebase.  While running off of the Enzo mercurial repository is in general
+codebase.  While running off of the Enzo GitHub repository is in general
```

Comment:
There are several places in the docs where you refer to 'GitHub' when you probably mean 'git'.  This is one of them. 
 Broadly speaking, you replaced 'mercurial' with 'GitHub' in many places where 'mercurial' is probably more correctly replaced with 'git', since you're talking about version control systems.

Reply:
Oops. We tried to avoid this but looks like we left some mistakes in there. I'll fix it.